.github/workflows/main.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         - name: Query dependencies
           run: |
             cp src/mlpack/bindings/R/mlpack/DESCRIPTION.in DESCRIPTION
-            Rscript -e "install.packages('remotes')" -e "saveRDS(remotes::dev_package_deps(dependencies = TRUE), 'depends.Rds')"
+            Rscript --verbose -e "install.packages('remotes')" -e "saveRDS(remotes::dev_package_deps(dependencies = TRUE), 'depends.Rds')"
```

Comment:
Do you like to keep the verbose output enabled?

Reply:
Only for debugging---I'll remove it once I get things working. :+1: